src/MaintenanceModeCommand.php

```diff
@@ -2,6 +2,154 @@
 
 namespace WP_CLI\MaintenanceMode;
 
```

Comment:
Imports should be added here:
```suggestion
use WP_CLI;
use WP_CLI_Command;
use WP_CLI\Utils;
use WP_Filesystem_Base;
use WP_Upgrader;

```

Reply:
Do we need to import `use WP_CLI\Utils;` ? as we already imported `WP_CLI` if `get_flag_value` is to be used without the namespace we can import the function. Let me know your thoughts.

Reply:
Yes, probably is more comprehensible to just stick with `use WP_CLI;` and the use `WP_CLI\Utils::some_function()...`.